Test1.js

```diff
@@ -1,6 +1,6 @@
 function test1 (){
 
-    return 
+    return "Hello world";
```

Comment:
I edited this syntaxes for testing purpose

Reply:
Thanks!!!